apps/api/src/app/events/services/workflow.queue.service.ts

```diff
@@ -156,6 +173,19 @@ export class WorkflowQueueService {
     return true;
   }
 
+  private async addDelayJob(data: JobEntity, options: JobsOptions): Promise<boolean> {
+    const isDelay = data.type === StepTypeEnum.DELAY && data.step.metadata.amount && data.step.metadata.unit;
```

Comment:
```suggestion
    const isValidDelayStep = data.type === StepTypeEnum.DELAY && data.step.metadata.amount && data.step.metadata.unit;
```

It does a bit more here than just checking the type. Wdyt?

Reply:
Sure. Good point. Will do the same for the digest function

Reply:
Sure. Good point. Will do the same for the digest function

Reply:
Sure. Good point. I will do the same for the digest function

Reply:
Sure. Good point. I will do the same for the digest function

Reply:
Sure. Good point. Will do the same for the digest function

Reply:
Sure. Good point. I will do the same for the digest function

Reply:
Sure. Good point. I will do the same for the digest function

Reply:
Sure. Good point. I will do the same for the digest function